### DISCUSSION TITLE
Update  1.12.2-1.21

### DISCUSSION THREAD

**Discussion by OleksandrChornyi2010** [Titlehhhh/Minecraft-DDOS-Free#59] [Ideas]
Update to 1.20.1 please!!!!!!!!!

**Comment by Titlehhhh** [comment]
At the moment, I found the time and made sure that the bot could log into the vanilla Minecraft server (online mode: false) version 1.21.1. The bot does not yet know how to write in chat, jump, run, etc. in this version. For now, I will try to add support for reading and writing basic packets (chat, movement, chunks), but in the future I plan to cover the entire Minecraft protocol from 1.12.2 to the  version of Minecraft and maybe even add bedrock edition.

After I add the main packets to McProtoNet, I will update the program itself so that the stress test bots can get new features
